tests/dataFromApisDB/setup_environment.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
Each file under `tests/` should really be under a folder called `src` or `source` or `app` or `digital-twin` or something that shows it is source code rather than test code